modules/tinymce/src/plugins/lists/test/ts/browser/BackspaceDeleteFromBlockIntoLiTest.ts

```diff
@@ -1,61 +1,60 @@
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/lists/Plugin';
+import Theme from 'tinymce/themes/silver/Theme';
 
-import ListsPlugin from 'tinymce/plugins/lists/Plugin';
-import SilverTheme from 'tinymce/themes/silver/Theme';
-
-UnitTest.asynctest('Browser Test: .RemoveTrailingBlockquoteTest', (success, failure) => {
-
-  ListsPlugin();
-  SilverTheme();
-
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyApis = TinyApis(editor);
-    const tinyActions = TinyActions(editor);
-
-    Pipeline.async({}, [
-      Log.stepsAsStep('TBA', 'Lists: backspace from p inside div into li', [
-        tinyApis.sFocus(),
-        tinyApis.sSetContent('<ul><li>a</li></ul><div><p><br /></p></div>'),
-        tinyApis.sSetCursor([ 1, 0, 0 ], 0),
-        tinyActions.sContentKeystroke(Keys.backspace(), { }),
-        tinyApis.sAssertContent('<ul><li>a</li></ul>')
-      ]),
-      Log.stepsAsStep('TBA', 'Lists: backspace from p inside blockquote into li', [
-        tinyApis.sFocus(),
-        tinyApis.sSetContent('<ul><li>a</li></ul><blockquote><p><br /></p></blockquote>'),
-        tinyApis.sSetCursor([ 1, 0, 0 ], 0),
-        tinyActions.sContentKeystroke(Keys.backspace(), { }),
-        tinyApis.sAssertContent('<ul><li>a</li></ul>')
-      ]),
-      Log.stepsAsStep('TBA', 'Lists: backspace from b inside p inside blockquote into li', [
-        tinyApis.sFocus(),
-        tinyApis.sSetContent('<ul><li>a</li></ul><blockquote><p><b><br /></b></p></blockquote>'),
-        tinyApis.sSetCursor([ 1, 0, 0, 0 ], 0),
-        tinyActions.sContentKeystroke(Keys.backspace(), { }),
-        tinyApis.sAssertContent('<ul><li>a</li></ul>')
-      ]),
-      Log.stepsAsStep('TBA', 'Lists: backspace from span inside p inside blockquote into li', [
-        tinyApis.sFocus(),
-        tinyApis.sSetContent('<ul><li>a</li></ul><blockquote><p><span class="x"><br /></span></p></blockquote>'),
-        tinyApis.sSetCursor([ 1, 0, 0, 0 ], 0),
-        tinyActions.sContentKeystroke(Keys.backspace(), { }),
-        tinyApis.sAssertContent('<ul><li>a</li></ul>')
-      ]),
-      Log.stepsAsStep('TBA', 'Lists: backspace from p into li', [
-        tinyApis.sFocus(),
-        tinyApis.sSetContent('<ul><li>a</li></ul><p><br /></p>'),
-        tinyApis.sSetCursor([ 1, 0 ], 0),
-        tinyActions.sContentKeystroke(Keys.backspace(), { }),
-        tinyApis.sAssertContent('<ul><li>a</li></ul>')
-      ])
-    ], onSuccess, onFailure);
-  }, {
+describe('Browser Test: .RemoveTrailingBlockquoteTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
     indent: false,
     plugins: 'lists',
     toolbar: '',
-    theme: 'silver',
     base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  }, [ Plugin, Theme ]);
+
+  it('TBA: Lists: backspace from p inside div into li', () => {
+    const editor = hook.editor();
+    editor.focus();
```

Comment:
Focus is called every test here, so we should move that out into something like this:
```
beforeEach(() => {
  hook.editor().focus();
});
```

- [ ] Move focus out into  `beforeEach`